pom.xml

```diff
@@ -102,6 +103,20 @@
             </configuration>
           </execution>
         </executions>
+        <configuration>
+          <rules>
+            <requireUpperBoundDeps>
+              <excludes combine.children="append">
```

Comment:
As far as I can tell, if I don't include `combine.children`, then the exclusions which I have listed are ignored, and reported as enforcer rule failures.

Is there a better way to state it, without using `combine.children`?